sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v3/query.sql

```diff
@@ -0,0 +1,718 @@
+    ARRAY_AGG(environment.partner.distributor_channel RESPECT NULLS ORDER BY submission_timestamp)[
+      SAFE_OFFSET(0)
+    ] AS partner_distributor_channel,
+    ARRAY_AGG(environment.partner.partner_id RESPECT NULLS ORDER BY submission_timestamp)[
+      SAFE_OFFSET(0)
+    ] AS partner_id,
+    ARRAY_AGG(
+      environment.settings.attribution.campaign RESPECT NULLS
+      ORDER BY
+        submission_timestamp
+    )[SAFE_OFFSET(0)] AS attribution_campaign,
+    ARRAY_AGG(environment.settings.attribution.content RESPECT NULLS ORDER BY submission_timestamp)[
+      SAFE_OFFSET(0)
+    ] AS attribution_content,
+    ARRAY_AGG(
+      environment.settings.attribution.experiment RESPECT NULLS
+      ORDER BY
+        submission_timestamp
+    )[SAFE_OFFSET(0)] AS attribution_experiment,
+    ARRAY_AGG(environment.settings.attribution.medium RESPECT NULLS ORDER BY submission_timestamp)[
+      SAFE_OFFSET(0)
+    ] AS attribution_medium,
+    ARRAY_AGG(environment.settings.attribution.source RESPECT NULLS ORDER BY submission_timestamp)[
+      SAFE_OFFSET(0)
+    ] AS attribution_source,
+    ARRAY_AGG(environment.settings.attribution.ua RESPECT NULLS ORDER BY submission_timestamp)[
+      SAFE_OFFSET(0)
+    ] AS attribution_ua,
+    ARRAY_AGG(
+      environment.settings.default_search_engine_data.load_path RESPECT NULLS
+      ORDER BY
+        submission_timestamp
+    )[SAFE_OFFSET(0)] AS engine_data_load_path,
+    ARRAY_AGG(
+      environment.settings.default_search_engine_data.name RESPECT NULLS
+      ORDER BY
+        submission_timestamp
+    )[SAFE_OFFSET(0)] AS engine_data_name,
+    ARRAY_AGG(
+      environment.settings.default_search_engine_data.origin RESPECT NULLS
+      ORDER BY
+        submission_timestamp
+    )[SAFE_OFFSET(0)] AS engine_data_origin,
+    ARRAY_AGG(
+      environment.settings.default_search_engine_data.submission_url RESPECT NULLS
+      ORDER BY
+        submission_timestamp
+    )[SAFE_OFFSET(0)] AS engine_data_submission_url,
+    ARRAY_AGG(environment.system.apple_model_id RESPECT NULLS ORDER BY submission_timestamp)[
+      SAFE_OFFSET(0)
+    ] AS apple_model_id,
+    ARRAY_AGG(metadata.geo.city RESPECT NULLS ORDER BY submission_timestamp)[
+      SAFE_OFFSET(0)
+    ] AS city,
+    ARRAY_AGG(metadata.geo.db_version RESPECT NULLS ORDER BY submission_timestamp)[
+      SAFE_OFFSET(0)
+    ] AS db_version,
+    ARRAY_AGG(metadata.geo.subdivision1 RESPECT NULLS ORDER BY submission_timestamp)[
+      SAFE_OFFSET(0)
+    ] AS subdivision1,
+    ARRAY_AGG(metadata.isp.name RESPECT NULLS ORDER BY submission_timestamp)[
+      SAFE_OFFSET(0)
+    ] AS isp_name,
+    ARRAY_AGG(normalized_channel RESPECT NULLS ORDER BY submission_timestamp)[
+      SAFE_OFFSET(0)
+    ] AS normalized_channel,
+    ARRAY_AGG(normalized_country_code RESPECT NULLS ORDER BY submission_timestamp)[
+      SAFE_OFFSET(0)
+    ] AS country,
+    ARRAY_AGG(norm.os(normalized_os) RESPECT NULLS ORDER BY submission_timestamp)[
```

Comment:
Can we pass `os` instead of `normalized_os`? Unfortunately we are having some issues with Mac/Darwin when we apply the function over the semi-normalized one.